src/app/content/components/Page/highlightManager.ts

```diff
@@ -172,6 +180,10 @@ export default (container: HTMLElement, getProp: () => HighlightProp) => {
         highlighter.highlight(pendingHighlight);
       }
 
+      getProp().highlights
+        .map(updateStyle(highlighter))
+      ;
```

Comment:
Semicolon probably should at the end on line 104

Reply:
i do this so i can add other array operations without worrying about where the semicolon goes, similar to why you put a comma on the end of the last element of a multiline array so you can add more easily and not have a dirty diff.

if its breaking someone's linter, i'll consider stopping, but i like it this way even if its a little non-standard